prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -3,7 +3,13 @@ plugin_paths = { "/prosody-plugins-custom" }
 
 VirtualHost "{{ .Env.XMPP_DOMAIN }}"
     {{ if .Env.ENABLE_AUTH | default "0" | toBool }}
+      {{ if .Env.ENABLE_LDAP_AUTH }}
+    authentication = "cyrus"
```

Comment:
Found a bug with authentication from jigasi. Need research how to split web (ldap auth) and XMPP_DOMAIN (internal auth)